Tests/RxSwift/MapManyTests.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  MapManyTests.swift
+//  RxSwiftExt
+//
+//  Created by Joan Disho on 06.05.18.
+//  Copyright © 2018 RxSwiftCommunity. All rights reserved.
+//
+
+import XCTest
+import RxSwift
+import RxTest
+
+class MapManyTests: XCTestCase {
+    fileprivate func runAndObserve<C: Collection>(_ sequence: Observable<C>) -> TestableObserver<C> {
```

Comment:
Same here like the sample project - please add something that tests mass initialization of a data model, like a struct. 

Thanks !